File: src/main/java/eschool/courses/model/Period.java
package eschool.courses.model;

import javax.jdo.annotations.*;

@PersistenceCapable
public class Period {
	@PrimaryKey
	@Persistent(valueStrategy=IdGeneratorStrategy.INCREMENT)	
	private long id;
	
	private String name;
	private int order;
	
	public Period() {}
	
	public Period(String name, int order) {
		this.name = name;
		this.order = order;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getOrder() {
		return order;
	}

	public void setOrder(int order) {
		this.order = order;
	}

	public long getId() {
		return id;
	}
}

File: src/main/scala/eschool/courses/model/Period.scala
package eschool.courses.model

import javax.jdo.annotations._
import jdo.Id

class Period extends Id[Long] {
  private[this] var _name: String = _
  private[this] var _order: Int = _
  
  def this(name: String, order: Int) = {
    this()
    _name = name
    _order = order
  }

  def name: String = _name
  def name_=(theName: String) { _name = theName }
  
  def order: Int = _order
  def order_=(theOrder: Int) { _order = theOrder }
}